### PR TITLE
fpsync: add page

### PR DESCRIPTION
**Pull request by floriankuek**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 1.6.1

### CODE REVIEW

File: pages/linux/fpsync.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# fpsync
+
+> Execute several synchronization processes locally or on several remote workers through SSH.
+> More information: <https://www.fpart.org/fpsync/>.
+
+- Recursivly syncronize a directory to another location:
+
+`fpsync -v {{/absolute/path/to/source/}} {{/absolute/path/to/destination/}}`
+
+- Recursivly syncronize a directory with the final pass. Enables rsync's --delete option with each synchronization job:
```
Comment (kbdharun) [inline pages/linux/fpsync.md:10]:
```suggestion
- Recursively synchronize a directory with the final pass (It enables rsync's `--delete` option with each synchronization job):
```

File: pages/linux/fpsync.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`fpsync -v -E {{/absolute/path/to/source/}} {{/absolute/path/to/destination/}}`
+
+- Recursivly syncronize a directory to a destination using 8 concurrent synchronization jobs:
```
Comment (kbdharun) [inline pages/linux/fpsync.md:14]:
```suggestion
- Recursively synchronize a directory to a destination using 8 concurrent synchronization jobs:
```

File: pages/linux/fpsync.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`fpsync -v -n 8 -E {{/absolute/path/to/source/}} {{/absolute/path/to/destination/}}`
+
+- Recursivly syncronize a directory to a destination using 8 concurrent synchronization jobs spread over two remote workers (machine1 and machine2):
```
Comment (kbdharun) [inline pages/linux/fpsync.md:18]:
```suggestion
- Recursively synchronize a directory to a destination using 8 concurrent synchronization jobs spread over two remote workers (machine1 and machine2):
```

File: pages/linux/fpsync.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`fpsync -v -n 8 -E -w login@machine1 -w login@machine2 -d {{/path/to/shared/directory}} {{/absolute/path/to/source/}} {{/absolute/path/to/destination/}}`
+
+- Recursivly syncronize a directory to a destination using 4 local workers, each one transferring at most 1000 files and 100 MB per synchronization job:
```
Comment (kbdharun) [inline pages/linux/fpsync.md:22]:
```suggestion
- Recursively synchronize a directory to a destination using 4 local workers, each one transferring at most 1000 files and 100 MB per synchronization job:
```

File: pages/linux/fpsync.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`fpsync -v -n 4 -f 1000 -s $((100 * 1024 * 1024)) {{/absolute/path/to/source/}} {{/absolute/path/to/destination/}}`
+
+- Recursivly syncronize any directories but exclude e.g. .snapshot* files. Options and values must be separated by a pipe character:
```
Comment (kbdharun) [inline pages/linux/fpsync.md:26]:
```suggestion
- Recursively synchronize any directories but exclude specific `.snapshot*` files (Note: options and values must be separated by a pipe character):
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

File: pages/linux/fpsync.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# fpsync
+
+> Execute several synchronization processes locally or on several remote workers through SSH.
+> More information: <https://www.fpart.org/fpsync/>.
+
+- Recursivly syncronize a directory to another location:
+
+`fpsync -v {{/absolute/path/to/source/}} {{/absolute/path/to/destination/}}`
```
Comment (spageektti) [inline pages/linux/fpsync.md:8]:
```suggestion
`fpsync -v {{/path/to/source/}} {{/path/to/destination/}}`
```
I don't think we need to say that the path is absolute, the `/` before is enough.

**Comment by floriankuek** [comment]
Thanks guys and sorry for the typos..

**Review by kbdharun — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Comment by spageektti** [comment]
Thanks for your contribution.

**Pull request merged**